pennylane/compiler/python_compiler/transforms/utils.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Registering transforms with plxpr to catalyst map"""
+
+import pennylane as qml
+from pennylane.devices.preprocess import null_postprocessing
+
+from .apply_transform_sequence import register_pass
+
+
+def xdsl_transform(_klass):
```

Comment:
I would personally prefer a different argument name. Maybe `pass`? Since we decorate passes with this?

Reply:
> It implies that it is not used inside the body of the function.

Agreed. Also what does klass mean?^^